### PR TITLE
translate 2021-09-27-SIG-Node-Spotlight

### PR DESCRIPTION
**Pull request by ldsdsy**

### CODE REVIEW

**Comment by tengqm** [comment]
@ldsdsy Please refer to other translations for examples on how we deal with frontmatter, the English text and how we split the translation into paragraphs.

Looking forward to more contributions from you.

**Comment by ldsdsy** [comment]
Thank you, I have made new changes, please review again。

File: content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+<!--
+## Introduction
+
+In Kubernetes, a _Node_ is a representation of a single machine in your cluster. [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node) owns that very important Node component and supports various subprojects such as Kubelet, Container Runtime Interface (CRI) and more to support how the pods and host resources interact. In this blog, we have summarized our conversation with [Elana Hashman (EH)](https://twitter.com/ehashdn) & [Sergey Kanzhelev (SK)](https://twitter.com/SergeyKanzhelev), who walk us through the various aspects of being a part of the SIG and share some insights about how others can get involved.
+-->
+
+## 介绍
+
+在 Kubernetes 中，一个 _Node_ 是你集群中的单个物理机。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md:28]:
```suggestion
在 Kubernetes 中，一个 _Node_ 是你的集群中的某台机器。
```

File: content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+[SIG Node](https://github.com/kubernetes/community/tree/master/sig-node) 这里拥有非常重要的 Node 组件并支持各种子项目，如 Kubelet, Container Runtime Interface (CRI) 以及 pod 和主机资源如何交互。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md:29]:
```suggestion
[SIG Node](https://github.com/kubernetes/community/tree/master/sig-node) 负责这一非常重要的
Node 组件并支持各种子项目，如 Kubelet、Container Runtime Interface (CRI) 以及
其他支持 Pod 和主机资源间交互的子项目。
```

File: content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+### Could you tell us a little about what SIG Node does?
+
+SK: SIG Node is a vertical SIG responsible for the components that support the controlled interactions between the pods and host resources. We manage the lifecycle of pods that are scheduled to a node. This SIG's focus is to enable a broad set of workload types, including workloads with hardware specific or performance sensitive requirements. All while maintaining isolation boundaries between pods on a node, as well as the pod and the host. This SIG maintains quite a few components and has many external dependencies (like container runtimes or operating system features), which makes the complexity we deal with huge. We tame the complexity and aim to continuously improve node reliability.
+-->
+## 我们的对话总结
+
+### 你能告诉我们一些关于 SIG Node 的工作吗？
+
+SK：SIG Node 是一个垂直 SIG ，负责支持 pod 和主机资源之间受控互动的组件。我们管理被安排到一个节点上的 pod 的生命周期。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md:43]:
```suggestion
SK：SIG Node 是一个垂直 SIG，负责支持 Pod 和主机资源之间受控互动的组件。
我们管理被调度到节点上的 Pod 的生命周期。
```

File: content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+这个 SIG 的重点是实现广泛的工作负载类型，包括具有硬件特性或性能敏感要求的工作负载。同时保持节点上 pod 之间的隔离界限，以及 pod 和主机的隔离界限。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md:44]:
```suggestion
这个 SIG 的重点是支持广泛的工作负载类型，包括具有硬件特性或性能敏感要求的工作负载。
同时保持节点上 Pod 之间的隔离边界，以及 Pod 和主机的隔离边界。
```

File: content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+EH: There are two kinds of SIGs: horizontal and vertical. Horizontal SIGs are concerned with a particular function of every component in Kubernetes: for example, SIG Security considers security aspects of every component in Kubernetes, or SIG Instrumentation looks at the logs, metrics, traces and events of every component in Kubernetes. Such SIGs don't tend to own a lot of code.
+
+Vertical SIGs, on the other hand, own a single component, and are responsible for approving and merging patches to that code base. SIG Node owns the "Node" vertical, pertaining to the kubelet and its lifecycle. This includes the code for the kubelet itself, as well as the node controller, the container runtime interface, and related subprojects like the node problem detector. 
+-->
+### 你能再解释一下 “SIG Node 是一种垂直 SIG” 的含义吗？
+
+EH：有两种 SIG：横向和垂直。横向 SIG 关注 Kubernetes 中每个组件的特定功能：例如，SIG Security 考虑 Kubernetes 中每个组件的安全方面，或者 SIG Instrumentation 关注 Kubernetes 中每个组件的日志、度量、跟踪和事件。
+这样的 SIG 并不太会拥有大量的代码。
+
+另一方面，垂直 SIG 拥有一个单一的组件，并负责批准和合并该代码库的补丁。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md:59]:
```suggestion
相反，垂直 SIG 拥有一个单一的组件，并负责批准和合并该代码库的补丁。
```

File: content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+
+In June, we ran our first bug scrub event to work through the backlog of issues filed against SIG Node, ensuring they were properly categorized. We closed over 130 issues over the course of this 48 hour global event, but as of writing we still have 333 open issues. 
+-->
+### 作为 issue 和 PR 数量第三大的 SIG，你们 SIG 是如何兼顾这么多工作的？
+
+EH：这归功于有组织性。当我在 2021 年 1 月增加对 SIG 的贡献时，我发现自己被大量的 PR 和 issue 淹没了，不知道该从哪里开始。
+我们已经在 CI 子项目板上跟踪与测试有关的 issue 和 PR 请求，但这缺少了很多 bug 修复和功能工作。
+因此，我开始为我们剩余的 PR 建立一个分流板，这使我能够根据状态和采取的行动对其进行分类，并为其他贡献者记录它的用途。
+在过去的两个版本中，我们关闭或合并了超过 500 个 issue 和 PR 。Kubernetes devstats 显示，我们的速度因此而大大提升。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md:85]:
```suggestion
在过去的两个版本中，我们关闭或合并了超过 500 个 issue 和 PR。Kubernetes devstats 显示，我们的速度因此而大大提升。
```

File: content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+<!--
+### Why should new and existing contributors consider joining SIG Node?
+
+SK: Being a SIG Node contributor gives you skills and recognition that are rewarding and useful. Understanding under the hood of a kubelet helps architecting better apps, tune and optimize those apps, and gives leg up in issues troubleshooting. If you are a new contributor, SIG Node gives you the foundational knowledge that is key to understanding why other Kubernetes components are designed the way they are. Existing contributors may benefit as many features will require SIG Node changes one way or another. So being a SIG Node contributor helps building features in other SIGs faster.
+
+SIG Node maintains numerous components, many of which have dependency on external projects or OS features. This makes the onboarding process quite lengthy and demanding. But if you are up for a challenge, there is always a place for you, and a group of people to support. 
+-->
+### 为什么新的和现有的贡献者应该考虑加入 Node 兴趣小组呢？
+
+SK：作为 SIG Node 的贡献者会带给你有意义且有用的技能和认可度。了解 Kubelet 的内部结构有助于构建更好的应用程序，调整和优化这些应用程序，并在 issue 的解决上获得优势。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md:98]:
```suggestion
SK：作为 SIG Node 的贡献者会带给你有意义且有用的技能和认可度。
了解 Kubelet 的内部结构有助于构建更好的应用程序，调整和优化这些应用程序，并在 issue 排查上获得优势。
```

File: content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+
+To come up to speed on any open source code base, there are two strategies you can take: start by exploring a particular issue deeply, and follow that to expand the edges of your knowledge as needed, or briefly review as many issues and change requests as you possibly can to get a higher level picture of how the component works. Ultimately, you will need to do both if you want to become a Node reviewer or approver.
+
+[Davanum Srinivas](https://twitter.com/dims) and I each ran a cohort of group mentoring to help teach new contributors the skills to become Node reviewers, and if there's interest we can work to find a mentor to run another session. I also encourage new contributors to attend our Node CI Subproject meeting: it's a smaller audience and we don't record the triage sessions, so it can be a less intimidating way to get started with the SIG. 
+-->
+###  你是如何帮助新手贡献者开始工作的？
+
+EH：在 SIG Node 的起步工作可能是令人生畏的，因为有太多的工作要做，我们的 SIG 会议非常大，而且很难找到一个开始的地方。
+
+我总是鼓励新手贡献者在他们已经有一些投入的方向上更进一步。在 SIG Node 中，这可能意味着自愿帮助修复一个只影响到你个人的 bug ，或者按优先级去分流你关心的 bug 。
```
Comment (tengqm) [inline content/zh/blog/_posts/2021-09-27-SIG-Node-Spotlight/index.md:119]:
```suggestion
我总是鼓励新手贡献者在他们已经有一些投入的方向上更进一步。
在 SIG Node 中，这可能意味着自愿帮助修复一个只影响到你个人的 bug，或者按优先级去分流你关心的 bug。
```

**Comment by ldsdsy** [comment]
I've reworked it, please

**Pull request merged**